src/main/kotlin/com/celuveat/member/adapter/in/rest/SocialLoginController.kt

```diff
@@ -34,7 +40,7 @@ class SocialLoginController(
         return LoginResponse.from(token)
     }
 
-    @GetMapping("/login/{socialLoginType}/url")
+    @GetMapping("/{socialLoginType}/url")
```

Comment:
사소한건데
회원탈퇴는 withdraw가 먼저 오고
이건 url이 나중에 오는 구조인데 따로 이유 있어??

Reply:
이유는 없고... 디테일의 부족!
"/withdraw/{socialLoginType}" 이 형식에 맞게 수정하는게 나을 거 같아!
```suggestion
    @GetMapping("url/{socialLoginType}")
```